templates.go

```diff
@@ -346,6 +350,12 @@ func sunAngleToTimeOfDay(angle int) string {
 	return t.Format("15:04")
 }
 
+func anonymiseDriverGUID(guid string) string {
```

Comment:
this function is a duplicate of `func AnonymiseDriverGUID`? in the template funcs above you can totally just assign `funcs["anonymiseDriverGUID"] = AnonymiseDriverGUID`